src/components/layout/tables/NameTable/NameTable.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+
+import {
+  CalendarTimeIcon,
+  LinkIcon,
+  PriceTagIcon,
+  RefreshAlertIcon,
+} from '../../../icons';
+import Paginator from '../../../inputs/Paginator/Paginator';
+import './styles.css';
+
+function NameTable() {
```

Comment:
`columns` would be a good prop here and each would have an `icon`

e.g.
```
function NameTable(columns: { name: string, icon: any })
```
and then do something like
```
{
   <thead>
         <tr className="assets-table-header">
           {Object.map((columns: { name: string, icon?: any }) => 
                return (
                  <td className="assets-table-item">
                        {name}
                  </td>
                 )
              })
           }
           </tr>
        </thead>
 ....
```

Reply:
also - generally table's can be tough. if you're finding it difficult to create one, we can look at third-party libs that provide customize tables.